src/Engine/StateTrackers/DamageDealtToIncomeConverter.java

```diff
@@ -57,10 +56,10 @@ private void profitize(UnitDelta attacker, UnitDelta defender)
 
   private static int calculateProfit(UnitDelta delta, double myIncomeRatio)
   {
-    int healthLoss = delta.getHealthDamage();
+    int healthLoss = delta.getHPDamage();
     int unitCost = delta.unit.getCost();
     // Do the necessary math, then round to the nearest int.
-    int income = (int) (healthLoss * (unitCost / UnitModel.MAXIMUM_HEALTH) * myIncomeRatio + 0.5);
+    int income = (int) (healthLoss * (unitCost / 10) * myIncomeRatio + 0.5);
```

Comment:
Why change this back to `10` instead of `UnitModel.MAXIMUM_HEALTH`?.

Reply:
... or an otherwise more correct non-magic named quantity?

Reply:
`MAXIMUM_HEALTH` is 100.
I guess we could *also* have `MAXIMUM_HP`, but that doesn't exactly align with all the usecases we have for "10".

I am roughly equally annoyed by long variable names and magic numbers, and naming all the different reasons we'd want to use "10" is hard, so I default to lazy.